itest/psbt_test.go

```diff
@@ -1022,13 +1032,38 @@ func testMultiInputPsbtSingleAssetID(t *harnessTest) {
 
 	ConfirmAndAssertOutboundTransfer(
 		t.t, t.lndHarness.Miner.Client, primaryTapd, sendResp,
-		genInfo.AssetId, []uint64{0, 4000}, 1, 2,
+		genInfo.AssetId, []uint64{changeAmt, sendAmt}, 1, 2,
 	)
 
 	AssertNonInteractiveRecvComplete(t.t, secondaryTapd, 2)
 
-	t.Logf("Two separate send events complete, now attempting to send " +
-		"back the full amount in a single multi-input PSBT send event")
+	// Third of three send events from primary (minting) node to the
+	// secondary node.
+	sendAmt = uint64(3000)
+	changeAmt = uint64(0)
+
+	addr, err = secondaryTapd.NewAddr(
```

Comment:
If the goal is to just create three different on-chain outputs, we _could_ use a multi-send (`sendAssetsToAddr()` supports multiple receiver addresses). Would shorten the code a bit.

Reply:
Using `sendAssetsToAddr` I ran into problems reusing an address. (Not all output script keys were unique.) I'll investigate and try to get it working but I'll leave this PR as is for now.

Reply:
Why would address reuse be necessary? IIUC each call to `NewAddr` should be returning addresses with unique script keys.

Reply:
I might try again without address re-use. Thanks for pointing out the unique script key property. I'll look into that.